ambry-api/src/main/java/com/github/ambry/server/StoreManager.java

```diff
@@ -106,4 +108,18 @@ public interface StoreManager {
    * @return {@code true} if disabling was successful. {@code false} if not.
    */
   boolean controlCompactionForBlobStore(PartitionId id, boolean enabled);
+
+  /**
+   * Get the primary {@link ClusterParticipant} for the cluster.
+   * @return the primary {@link ClusterParticipant} for the cluster.
+   */
+  ClusterParticipant getPrimaryClusterParticipant();
```

Comment:
I see a secondary participant was also mentioned somewhere. Are we sure that we don't need it here?

Reply:
Afaik, yea, primary is what we need. 

```
// In most cases, there should be only one participant in the clusterParticipants list. If there are more than one
// and some components require sole participant, the first one in the list will be primary participant.
```